data/ecosystem/vendors.yaml

```diff
@@ -1,3 +1,9 @@
+- name: Apica
+  nativeOTLP: true
+  url: https://docs.apica.io/technologies/ascent-with-opentelemetry
```

Comment:
looks like this URL suits the requirements better as it calls out how one can send otlp to your backend: 

https://docs.apica.io/integrations/list-of-integrations/opentelemetry